src/java/org/sakaiproject/feedback/tool/entityproviders/FeedbackEntityProvider.java

```diff
@@ -30,6 +33,7 @@
 import net.tanesha.recaptcha.ReCaptcha;
 import net.tanesha.recaptcha.ReCaptchaFactory;
 import net.tanesha.recaptcha.ReCaptchaResponse;
+import org.sakaiproject.util.ResourceLoader;
```

Comment:
Sorry, ResourceLoader is another one that should be in SakaiProxy as it pulls in the component manager.